src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/ValueTask.cs

```diff
@@ -114,6 +111,40 @@ private ValueTask(object? obj, short token, bool continueOnCapturedContext)
             _continueOnCapturedContext = continueOnCapturedContext;
         }
 
+        /// <summary>Gets a task that has already completed successfully.</summary>
+        public static ValueTask CompletedTask => default;
```

Comment:
does default value mean completed?

Reply:
nevermind, I am seeing you already have a test case asserting default is completed task.

Reply:
Yes, a `default(ValueTask)` is successfully completed.